package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "node-firefox-forward-ports",
+  "version": "1.0.0",
+  "description": "Find attached FirefoxOS devices using ADB",
+  "main": "index.js",
+  "dependencies": {
+    "adbkit": "^2.1.6",
+    "es6-promise": "^2.0.1",
+    "portfinder": "^0.4.0"
+  },
+  "devDependencies": {
+    "gulp": "^3.8.10",
+    "gulp-jscs": "^1.3.1",
```

Comment:
do you *actually* need all these gulp-* dependencies or are them copy paste remains? IIRC they are included by the `build-tools` module and you should not need to install them in the top module